contract/wizards/contract_manually_create_invoice.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="contract_manually_create_invoice_form_view">
+        <field name="model">contract.manually.create.invoice</field>
+        <field name="arch" type="xml">
+            <form string="Contract Manually Create Invoice">
+                <group>
+                    <group>
+                        <field name="invoice_date"/>
+                    </group>
+                    <group>
+                        <button name="action_show_contract_to_invoice" type="object"
+                                class="btn-link"
+                                attrs="{'invisible': [('contract_to_invoice_count', '=', 0)]}">
+                            <field name="contract_to_invoice_count"/>
+                            <span attrs="{'invisible': [('contract_to_invoice_count', '&gt;', 1)]}">
+                                contract to invoice
+                            </span>
+                            <span attrs="{'invisible': [('contract_to_invoice_count', '&lt;', 1)]}">
+                                contracts to invoice
+                            </span>
+                        </button>
+                    </group>
+                </group>
+                <footer>
+                    <button name="create_invoice"
+                            string="Create Invoices"
+                            class="btn-primary"
+                            type="object"/>
+                    <button string="Cancel"
+                            class="btn-default"
+                            special="cancel"/>
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window"
+            id="contract_manually_create_invoice_act_window">
+        <field name="name">Manually Invoice Contracts</field>
+        <field name="res_model">contract.manually.create.invoice</field>
+        <field name="view_mode">form</field>
+        <field name="context">{}</field>
+        <field name="target">new</field>
+    </record>
+
+    <record model="ir.ui.menu" id="contract_manually_create_invoice_menu">
+        <field name="name">Manually Invoice Contracts</field>
+        <field name="parent_id" ref="account.menu_finance"/>
+        <field name="groups_id" eval="[(6, 0, [ref('account.group_account_manager')])]"/>
+        <field name="action"
+               ref="contract_manually_create_invoice_act_window"/>
+        <field name="sequence" eval="16"/>
```

Comment:
Does it land in a reasonably logical place in the menu?

Reply:
![1](https://user-images.githubusercontent.com/12665409/71911223-a0620580-3173-11ea-90c0-405f2f76977e.png)

Reply:
Uhm, it's a long text for being in main menu. Maybe split it in 2 actions, one for customer and one for suppliers, and put them into their respective submenus?

Reply:
done, https://github.com/OCA/contract/pull/414/commits/91f537789d867b6bb0d80cb6ba5a4c31289d42d4

![1](https://user-images.githubusercontent.com/12665409/72610427-9e443780-3927-11ea-8ceb-a65001b86674.png)